README.md

```diff
@@ -89,10 +76,64 @@ Then, visit http://127.0.0.1:8000/. This page displays a "Congratulations!" mess
   - Fully integrated with Django's authentication framework.  
   - Supports native user management features like creating users and sessions.
 
+## Limitations of django-mongodb-backend
 
-- **Management Commands**  
-    
-  - Use commands like `migrate`, `makemigrations`, `flush`, `sqlmigrate`, many more.
+- Database Variables `ATOMIC_REQUESTS`, `AUTOCOMMIT`, `CONN_HEALTH_CHECKS`, `TIME_ZONE` not supported
+- No support for GeoDjango
+- Functions such as `Chr`, `ExtractQuarter`, `MD5`, `Now`, `Ord`, `Pad`, `Repeat`, `Reverse`, `Right`, `SHA1`, `SHA224`, `SHA256`, `SHA384`, `SHA512`, and `Sign`.
+- The `tzinfo` parameter of the `Trunc` database functions does not work properly because MongoDB converts the result back to UTC.
+- Schema Validation is not enforced. Refer to MongoDB documentation for how to enforce schema validation.
```

Comment:
Nobody said it was supported. Give a more helpful pointer if this really needs to be said?

Reply:
> Nobody said it was supported. Give a more helpful pointer if this really needs to be said?

I'd definitely prefer linking to unsupported features rather than listing them here.